src/services/doppler-legacy-client.ts

```diff
@@ -244,6 +259,7 @@ export function mapHeaderDataJson(json: any) {
       lang: json.user.lang,
       nav: (json.user.nav && json.user.nav.map(mapNavEntry)) || [],
       plan: mapPlanEntry(json.user.plan),
+      sms: mapSmsEntry(json.user?.sms),
```

Comment:
How about avoiding all inner mapping if there are no properties? Alongside with changes in `HeaderUserMenu` component line 72. 
```suggestion
      sms: Object.keys(Json.user.sms).length? mapSmsEntry(json.user.sms): {},
```